src/cli.js

```diff
@@ -160,7 +183,7 @@ module.exports = function (inputArgs, cb) {
 };
 
 function getSubCommand(args, cmd) {
-    if(cmd === 'platform' || cmd === 'platforms' || cmd === 'plugin' || cmd === 'plugins' || cmd === 'telemetry') {
+    if(cmd === 'platform' || cmd === 'platforms' || cmd === 'plugin' || cmd === 'plugins' || cmd === 'telemetry' || cmd === 'config') {
```

Comment:
This is absolutely going to be a merge conflict with #270 
https://github.com/apache/cordova-cli/pull/270/files#diff-d9d9214113dea1c364abd4ba3c70be11R317